sphire/__init__.py

```diff
@@ -185,8 +185,9 @@ def versionGE(cls, version):
     @classmethod
     def runCryolo(cls, protocol, program, args, cwd=None, useCpu=False):
         """ Run crYOLO command from a given protocol. """
-        fullProgram = '%s %s && %s' % (cls.getCondaActivationCmd(),
-                                       cls.getCryoloEnvActivation(useCpu), program)
+        envAct = cls.getCryoloEnvActivation(useCpu)
+        condaAct = cls.getCondaActivationCmd() if envAct.startswith('conda activate') else ''
```

Comment:
this will break in my case:
`CRYOLO_ENV_ACTIVATION = module load cuda/11.6 && conda activate cryolo-1.9.7`

Reply:
You can manually prepend the conda activation or put a line in a bash file with all three commands.  The condition here to avoid pretending conda env is somehow arbitrary, but I think it is necessary. I'm open to other suggestions on how to prevent the pre-prending.

Reply:
Maybe we can define a "loader" variable that takes preference if it exists and nothing is added. If it does not exist, things remain the same as now. What do you think? The same idea could be used in other plugins

Reply:
can we replace `envAct.startswith('conda activate')` with `'conda activate' in envAct`, so both of us are happy? :) I understand that you dont want to source the conda itself if "conda activate" is not present (e.g. hen loading modules or sourcing shell files etc)

Reply:
I suggest to revert this change and address new var business in a separate PR, because it will require more work

Reply:
I agree. I will do that.